src/alfrescoApi.ts

```diff
@@ -233,6 +233,10 @@ export class AlfrescoApi implements Emitter {
      * */
     login(username: string, password: string): Promise<any> {
 
+        if (password === undefined || password === null || username === undefined || username === null) {
```

Comment:
maybe worth checking for empty string as well ?

Reply:
Yep I agree. it was part of the acceptance criteria :p